ui_testing/testcases/header_tests/test08_usermanagement.py

```diff
@@ -238,4 +227,289 @@ def test010_update_user_role_with_save_cancel_btn(self, save):
                 ' + Assert {} (current_role) == {} (user_role)'.format(current_role, user_role))
             self.assertEqual(current_role, user_role)
 
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test010_update_user_email_with_save_cancel_btn(self, save):
+        """
+        User management: I can update user email with save & cancel button
+        LIMS-6397
+        :return:
+        """
+        self.base_selenium.click(element='header:user_management_button')
+        # open random user in the edit mode
+        self.header_page.get_random_user()
+        user_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + user_url : {}'.format(user_url))
+        self.order_page.sleep_tiny()
+        current_email = self.header_page.get_user_email()
+        self.header_page.set_user_email((self.header_page.generate_random_email()))
+        new_email = self.header_page.get_user_email()
+        if 'save_btn' == save:
+            self.header_page.save(save_btn='user_management:save_btn')
+        else:
+            self.header_page.cancel(force=True)
+
+            self.base_selenium.get(
+                url=user_url, sleep=self.base_selenium.TIME_MEDIUM)
+
+        user_email = self.header_page.get_user_email()
+        if 'save_btn' == save:
+            self.base_selenium.LOGGER.info(
+                    ' + Assert {} (new_email) == {} (user_email)'.format(new_email, user_email))
+            self.assertEqual(new_email, user_email)
+        else:
+            self.base_selenium.LOGGER.info(
+                    ' + Assert {} (current_email) == {} (user_email)'.format(current_email, user_email))
+            self.assertEqual(current_email, user_email)
+
+    def test011_validation_user_name_email_fields(self):
+        """
+        Header: User management: Make sure when the user update name & email then press on save button,
+        red border display and he can't save
+        LIMS-6121
+        :return:
+            """
+        # from the create mode it will redirect me to the active table
+        self.base_selenium.click(element='header:user_management_button')
+        self.header_page.get_random_user()
+        self.header_page.clear_user_name()
+        self.header_page.clear_user_email()
```

Comment:
Test case says, Update not clear. Are they the same?

Reply:
@0xIslamTaha I mean update the field by clear it

Reply:
@deenaahmed hmm, I think they should be different 
1- Update the current data, should be updated
2- Empty required fields should raise an error.
correct me if I'm wrong